modules/ROOT/pages/install-prereqs.adoc

```diff
@@ -101,8 +101,8 @@ If you plan to enable inbound traffic, evaluate your network's traffic to determ
 
 Runtime Fabric on VMs / Bare Metal requires one of the following operating systems:
 
-* Red Hat (RHEL) v7.4, v7.5, v7.6, v7.7, v7.8, v7.9, v8.0, v8.1, v8.2, v8.3
-* CentOS v7.4, v7.5, v7.6, v7.7, v7.8, v7.9, v8.0, v8.1, v8.2, v8.3
+* Red Hat (RHEL) v7.4, v7.5, v7.6, v7.7, v7.8, v7.9, v8.0, v8.1, v8.2, v8.3, v8.4
```

Comment:
Do we have to mention the appliance version for 8.4 support (Similar to what we say for Ubuntu 18.04 support)?

Reply:
Yeah. I can do that

Reply:
added on a different PR